Framework/include/Baseline.h

```diff
@@ -285,6 +284,20 @@ class Baseline
         // Define QCD CR Selections
         //  -- common for all 3 channels for now !!!
         // -----------------------------------------
+         bool pass_qcdCR_0l = JetID                    && 
+                              passMETFilters           &&
+                              passMadHT                &&
+                              passNonIsoTrigger        &&
+                              passNonIsoTriggerMC      &&
+                              passElectronHEMveto      &&
+                              (runtype != "Data" || filetag.find("Data_SingleMuon") != std::string::npos) &&
+                              HT_NonIsoMuon_pt30 > 500 &&
+                              NNonIsoMuons == 1        &&
+                              NGoodMuons == 0          &&
+                              NGoodElectrons == 0      &&
+                              NGoodJets_pt30 >= 8      &&
+                              NGoodJets_pt45 >= 6;
+
         bool pass_qcdCR = JetID                    && 
```

Comment:
For amazing symmetry, can we rename this to `pass_qcdCR_1l` ? And likewise for line L348.